inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_selector_params.h

```diff
@@ -404,6 +404,7 @@ struct Params {
     std::string layerID;
     std::string forceImplementation;
     EngineInfo engineInfo;
+    std::string uniqueID;
```

Comment:
Is it used somewhere?

Reply:
It is used to pass the unique_id to the optional_params which is append unique_id to the end of KernelID. thanks